src/js/components/Button/StyledButton.js

```diff
@@ -142,6 +142,8 @@ const plainStyle = (props) => css`
 
 const activeButtonStyle = (props) => css`
   ${activeStyle}
+  ${props.activeIndicator &&
```

Comment:
It looks like we're adding `activeIndicator` to Button? If so, I think we're missing typescript, propTypes, unit tests, etc. Also, I'm wondering why we need this at the Button API level. I would expect that the presence of the `active` property would be sufficient. In other words, if I indicate that a Button is `active`, then I would expect the Button to indicate that visually some how.

Reply:
Details about isolated use in select in the code comment here: https://github.com/grommet/grommet/pull/6243/files#r927979426

Reply:
open to other ways of achieving the functionality, but the functionality is needed